HttpSignature.md

```diff
@@ -1,40 +1,160 @@
-# HTTP-Sig Authentication for SoLiD
+# HttpSig Authentication for SoLiD
 
-[Signing HTTP Messages](https://datatracker.ietf.org/doc/draft-cavage-http-signatures/) (henceforth `HTTP-Sig`) is an IETF RFC Draft that has been developing since 2013 for signing and authenticating HTTP messages. It has the advantage of being very simple and being  specified directly at the HTTP layer. As a result there are a [large number of implementations](https://github.com/w3c-dvcg/http-signatures/issues/1), and is already widely used (todo: link).  
+## Summary
 
-The protocol allows a client to authenticate by signing a number of HTTP headers with it's private key. In order for the server to be able to verify this signature it needs to know the matching public key. This information must be transmitted by the client in the form of an opaque string known as a `keyId` (see [§2.1.1 keyId](https://tools.ietf.org/html/draft-cavage-http-signatures-11#section-2.1.1)). This string must enable the server to look up the key. How this is done is not specified by the protocol.
+HttpSig is a simple but very efficient authentication protocol extending [Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01) RFC by defining &mdash;
+ * a `WWW-Authenticate: HttpSig` header the server can return with a 401 or 402 to the client
```

Comment:
I had forgotten that the draft-cavage had a `Signature` header for authentication. I'd be happy to use that too, but don't want to step on toes in case the intent is too different. Perhaps @msporny has a view on that?